gnucash/report/standard-reports/transaction.scm

```diff
@@ -214,14 +214,14 @@ options specified in the Options panels."))
 (define (sortkey-get-info sortkey info)
   (cdr (assq info (cdr (assq sortkey sortkey-list)))))
 
-(define (timepair-year tp)    (gnc:timepair-get-year tp))
```

Comment:
Will this commit work without the changes you have been discussing with @limitedAtonement on IRC recently ? If not I'd prefer to keep the timepair work on a separate branch in order to get your transaction report work merged in. It's becoming quite large and hard to follow for me :(
If these changes stand on their own, they can stay of course.

Reply:
Yes it does work without the time64 upgrade elsewhere. 

When this tr is merged in, the next work of applying time64 general upgrade will only require a couple of lines upgrading here.